src/main/resources/db/migration/V4__modify_comment_opinion_status_in_comment_table.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE comment
+MODIFY COLUMN comment_opinion_status
+    enum('POSITIVE', 'NEUTRAL', 'NEGATIVE', 'NOT_OPINION', 'NOT_CHECKED') default 'NOT_CHECKED' null
```

Comment:
z ciekawosci, co daje ten `null` na koncu?

A tak w ogole to dobre nazwy, ja mam zawsze z tym problem ;)

Reply:
Nie mam pojęcia. Tak było w V3 przy add column :D

Reply:
nie bylo moj drogi :)
<img width="1130" alt="Zrzut ekranu 2020-07-29 o 13 56 01" src="https://user-images.githubusercontent.com/42154043/88797341-7c175600-d1a3-11ea-97a2-fee94668753e.png">